ui/selectors/selectors.js

```diff
@@ -2470,10 +2470,7 @@ export function getIsDesktopEnabled(state) {
 export function getSnapsList(state) {
   const snaps = getSnaps(state);
   return Object.entries(snaps)
-    .filter(
-      ([_key, snap]) =>
-        !snap.preinstalled && snap.status !== SnapStatus.Installing,
-    )
+    .filter(([_key, snap]) => snap.status !== SnapStatus.Installing)
```

Comment:
Is this really related to this PR? I guess it's required to trigger the watch-only account creation from the Snaps menu, but do we want to have this as a new behavior?

Reply:
It is not necessary for the PR. I can remove it but then I will have to add to disable that selector in the manual testing steps.